examples/streaming/storm09/src/main/scala/io/gearpump/streaming/examples/storm/Storm09KafkaTopology.scala

```diff
@@ -24,12 +24,12 @@ import backtype.storm.topology.TopologyBuilder
 import storm.kafka.{ZkHosts, SpoutConfig, KafkaSpout}
 import storm.kafka.bolt.KafkaBolt
 
-object StormKafkaTopology extends App {
+object Storm09KafkaTopology extends App {
   if (args.length < 7) {
     val usage =
       """
         | Usage:
-        |    StormKafkaTopology topologyName spoutNum boltNum
+        |    Storm09KafkaTopology topologyName spoutNum boltNum
         |         kafkaSpoutTopic kafkaBoltTopic zookeeperConnect brokerList
```

Comment:
This help info is not helpful.
For example,  what is kafkaSpoutTopic, kafkaBoltTopic ?

How to pass zookeeperConnect  or brokerList?